### PR TITLE
docs: add Japanese translated docs

### PR DESCRIPTION
**Pull request by eltociear**
## Description
Add Japanese translated docs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,8 @@
 English |
 [Русский](README_ru-RU.md) |
 [简体中文](README_zh-CN.md) |
-[正體中文](README_zh-TW.md)
+[正體中文](README_zh-TW.md) |
+[日本語](README_ja-JP.md)
```
Comment (ouuan) [inline README.md:19]: The languages are sorted by the language code, so put it between English and Russian.

**Comment by coder3101** [comment]
You can also add Japanese as application language. Refer #1131

**Comment by coder3101** [comment]
> In China, we usually put a space between English words and Chinese characters, do you add these spaces in Japan?

From ChatGPT:

In Japan, it is common to use a space between English words and Japanese characters when writing in mixed script, similar to the way it is done in China. This is done to make the text more legible and easier to read.

File: README.md (hunk 1 above)
Comment (eltociear) [reply README.md]: @ouuan Thanks! Fixed it.

**Review by ouuan — APPROVED** [review]
LGTM, thanks!

**Comment by ouuan** [comment]
@all-contributors add @eltociear for translations.

**Comment by ouuan** [comment]
@all-contributors add @eltociear for translation.

**Pull request merged**